text/70_performance_testing.md

```diff
@@ -14,13 +14,13 @@ As mentioned above, defining performance for a particular system means determini
 
 Just like service-oriented indicators, there are two main subcategories in efficiency-oriented testing.  __Throughput__ measures how many events can the system handle in a given amount of time, such as how many users can log in at the same time, or how many packets can a system route in 30 seconds.  __Utilization__ measures what percentage or absolute amount of computing resources are used to perform a given task.  For example, how much disk space does a database take to store a specified table?  How much RAM is necessary when sorting it?
 
-Service-oriented indicators are similar to black-box testing in that they measure the system from an external perspective, without determining the cause of the problem precisely.  They are blunt instruments from broadly effective in finding where problems should be looked for more closely.  Efficiency-oriented indicators are similar to white-box testing in that they require knowledge of the system, as well as general technical knowledge.  Oftentimes, you can use service-oriented indicators to determine general areas where performance might be an issue to users, and then use efficiency-oriented indicators to nail down exactly what is causing the problem.  Long response times from the perspective of the user may result from physical RAM being used up and much of the information needing to be swapped in from disk.  Service-oriented testing would catch the former, but efficiency-oriented testing would be able to figure out the latter.
+Service-oriented indicators are similar to black-box testing in that they measure the system from an external perspective, without determining the cause of the problem precisely.  As blunt instruments, they are broadly effective in finding where problems should be looked for more closely.  Efficiency-oriented indicators are similar to white-box testing in that they require knowledge of the system, as well as general technical knowledge.  Oftentimes, you can use service-oriented indicators to determine general areas where performance might be an issue to users, and then use efficiency-oriented indicators to nail down exactly what is causing the problem.  Long response times from the perspective of the user may result from physical RAM being used up and much of the information needing to be swapped in from disk.  Service-oriented testing would catch the former, but efficiency-oriented testing would be able to figure out the latter.
```

Comment:
Original: "They are blunt instruments from broadly effective in finding..."

Proposed: "As blunt instruments, they are broadly effective in finding..."